docs/REQUIREMENTS.md

```
@@ -0,0 +1,68 @@
+
+  // 文字列化して出力
+  console.log(html.toString());
+  // または `${html}` で評価
+  ```
+
+### ② CLIツールとしての利用
+
+- コマンドライン（ターミナル）から `.md` ファイルを指定して `.html` ファイルを生成できること。
+- コマンドライン引数でカスタムCSSファイルやタイトル等を指定できること。
+  ```bash
+  $ npx md2html input.md -o output.html --css ./custom.css
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)